src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/HaFlowExtended.groovy

```diff
@@ -92,4 +106,78 @@ class HaFlowExtended {
                                boolean useTraffgenPorts = true, List<Integer> busyEndpoints = []) {
         return new HaFlowBuilder(swT, northboundV2, topologyDefinition, useTraffgenPorts, busyEndpoints)
     }
+
+    HaFlowUpdatePayload convertToUpdateRequest() {
+
+        String diverseLink = diverseWithFlows.isEmpty() ?
```

Comment:
In this particular case a 'ladder' of `if-else` is preferable than three ternary operators. Also, I'd move it to HaFlowExtended's method with a clear name, like `getAnyDiverseFlow()` or whatever you are trying to get here.

Reply:
Done.